pints/_optimisers/__init__.py

```diff
@@ -598,7 +643,12 @@ def run(self):
             print('Unexpected termination.')
             print('Current best score: ' + str(fbest))
             print('Current best position:')
-            for p in self._optimiser.xbest():
+            # Inverse transform search parameters
```

Comment:
You can stick some newlines around this, and do it outside of this if block (and only once!)

Reply:
Added some newlines around this. But I don't see how this can be done outside the except block (which includes KeyboardInterrupt!)...